### PR TITLE
rename opengpt by rungpt

### PR DESCRIPTION
**Pull request by huangkaipeng4399**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,11 +4,11 @@
 from setuptools import find_packages, setup
 
 if sys.version_info < (3, 8, 0):
-    raise OSError(f'OpenGPT requires Python >=3.8, but yours is {sys.version}')
+    raise OSError(f'RunGPT requires Python >=3.8, but yours is {sys.version}')
 
 try:
-    pkg_name = 'open-gpt-torch'
-    libinfo_py = path.join(path.dirname(__file__), 'open_gpt', '__init__.py')
+    pkg_name = 'run-gpt-torch'
```
Comment (jemmyshin) [inline setup.py:10]: maybe we can use `run-gpt` instead of `run-gpt-torch`

**Comment by jemmyshin** [comment]
@numb3r3 I think we can release 0.1 or even 1.0 after this pr is merged

**Comment by numb3r3** [comment]
@huangkaipeng4399 Does the example script works with `run_gpt`?

File: setup.py (hunk 1 above)
Comment (numb3r3) [reply setup.py:10]: @huangkaipeng4399 yes, the suffix `-torch` is only for passing the name validation of pypi. We can directly use `run-gpt` as the Python package name.

**Pull request merged**